bentoml/_internal/models/model.py

```diff
@@ -104,7 +128,8 @@ def from_fs(cls, item_fs: FS) -> "Model":
             logger.warning(f"Failed to import Model from {item_fs}.")
             raise BentoMLException("Failed to create Model because it was invalid")
 
-        res = cls(info.tag, item_fs, info)
+        # set _custom_objects to None for lazy loading
+        res = cls(info.tag, item_fs, info, None)
```

Comment:
Does the `None` not happen automatically with the default provided in the field definition?